assets/app/views/_tasks.html

```diff
@@ -19,7 +19,7 @@
             </ul>
           </div>
           <div ng-show="task.hasErrors">
-            <a href="javascript:;" ng-click="toggleExpand()">
+            <a href="javascript:;" ng-click="expanded = !expanded">
```

Comment:
where does `expanded` get defined? what limits its scope to this loop of `ng-repeat`?